diff -up ./azure/cli/core/telemetry.py.orig ./azure/cli/core/telemetry.py
--- ./azure/cli/core/telemetry.py.orig	2021-06-16 14:14:43.190950471 -0500
+++ ./azure/cli/core/telemetry.py	2021-06-16 14:14:46.576931599 -0500
@@ -442,7 +442,7 @@ def is_telemetry_enabled():
     from azure.cli.core.cloud import cloud_forbid_telemetry
     if cloud_forbid_telemetry(_session.application):
         return False
-    return _get_config().getboolean('core', 'collect_telemetry', fallback=True)
+    return _get_config().getboolean('core', 'collect_telemetry', fallback=False)
 
 
 # definitions
