diff -up ./setup.py.orig ./setup.py
--- ./setup.py.orig	2021-06-15 16:08:32.237803186 -0500
+++ ./setup.py	2021-06-15 16:08:40.344759109 -0500
@@ -82,7 +82,7 @@ setup(
     install_requires=[
         'msrest>=0.6.21',
         'azure-common~=1.1',
-        'azure-mgmt-core>=1.2.0,<2.0.0',
+        'azure-mgmt-core>=1.2.0',
     ],
     extras_require={
         ":python_version<'3.0'": ['azure-mgmt-nspkg'],
